### PR TITLE
CLI-624: move alerts out of tab navigation into to settings

### PR DESCRIPTION
**Pull request by mike-dydx**
## Links (dYdX Internal Use Only)
Linear Ticket: [CLI-624: move alerts out of tab navigation into to settings](https://linear.app/dydx/issue/CLI-624/move-alerts-out-of-tab-navigation-into-to-settings)

Figma Design: https://www.figma.com/design/mKevZOfE9nj6MZpiolKYW1/dYdX-%E2%80%BA-Mobile?node-id=9615-982&t=QjiTEfHMhHzJgIrk-4

<br/>

## Description / Intuition
- moves in-app alerts to settings L2 screen
- removes news from alerts/news screen (also some renaming to reflect non-news behavior)
- moves earn/vaults tab to L1 tab navigation screen 
- earn/vaults tab icon replaces news/alerts icon
- fixes issue where tab bar gradient flashes a color when app theme changes

<br/>

## Before/After Screenshots or Videos

| Desc | After |
|--------|-------|
| hidden - flag off |  |
| showing - flag on |  |
| smaller device |  |
| new alerts screen without news |  |
| L1 vaults screen |  |
| light/dark mode switching and tab bar scroll behind | <video src="https://github.com/user-attachments/assets/6b8114b5-10d1-4088-b67f-13bf4c92c751" width=60%> |

<br/>

### Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring or Technical Debt
- [ ] Documentation update
- [ ] Other (please describe: )

### CODE REVIEW

File: dydx/dydxPresenters/dydxPresenters/_v4/MarketInfo/dydxMarketInfoViewBuilder.swift [hunk 1]
```diff
@@ -19,7 +19,7 @@ public class dydxMarketInfoViewBuilder: NSObject, ObjectBuilderProtocol {
     public func build<T>() -> T? {
         let presenter = dydxMarketInfoViewPresenter()
         let view = presenter.viewModel?.createView() ?? PlatformViewModel().createView()
-        let viewController = dydxMarketInfoViewController(presenter: presenter, view: view, configuration: .nav)
+        let viewController = dydxMarketInfoViewController(presenter: presenter, view: view, configuration: .default)
```
Comment (mike-dydx) [inline dydx/dydxPresenters/dydxPresenters/_v4/MarketInfo/dydxMarketInfoViewBuilder.swift:22]: nav is exactly the same as default

File: dydx/dydxViews/dydxViews/SwiftUIHosting/HostingViewController.swift [hunk 2]
```diff
@@ -33,7 +33,6 @@ public struct HostingViewControllerConfiguration {
     public static let `default` = HostingViewControllerConfiguration(ignoreSafeArea: false, fixedHeight: nil, gradientTabbar: false)
     public static let ignoreSafeArea = HostingViewControllerConfiguration(ignoreSafeArea: true)
     public static let tabbarItemView = HostingViewControllerConfiguration(ignoreSafeArea: false, fixedHeight: nil, gradientTabbar: true)
-    public static let nav = HostingViewControllerConfiguration(ignoreSafeArea: false, fixedHeight: nil, gradientTabbar: false, disableNavigationController: false)
```
Comment (mike-dydx) [inline dydx/dydxViews/dydxViews/SwiftUIHosting/HostingViewController.swift:36]: see line 21 to confirm nav was same as default

File: dydx/dydxViews/dydxViews/_v4/Alerts/Components/Alerts/dydxAlertsView.swift [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            
+            return VStack(spacing: 20) {
+                self.createHeader(parentStyle: style)
+                    .frame(height: 48)
+                    .padding([.leading, .trailing])
+
+                self.createAlertsView(parentStyle: style)
+            }
+            // offset for tab bar. comes from dydxV4TabBarBuilder's center button height
+                .padding(.bottom, 30)
```
Comment (mike-dydx) [inline dydx/dydxViews/dydxViews/_v4/Alerts/Components/Alerts/dydxAlertsView.swift:66]: this seems to be how we do it elsewhere 😞  unsure if there's a good solution right now to improve other than adding a comment

**Comment by ruixhuang** [comment]
The vault tab icon doesn't look like the same size as other tab icons.  I think we should increase its size.  Also, the icon looks like a "close" button.  

Screenshot 2024-09-18 at 14 55 26

**Review by ruixhuang — APPROVED** [review]

**Review by ruixhuang — APPROVED** [review]

**Pull request merged**